.goreleaser.yml

```diff
@@ -1,45 +1,52 @@
-# This is an example goreleaser.yaml file with some sane defaults.
+---
 # Make sure to check the documentation at http://goreleaser.com
 before:
   hooks:
     # you may remove this if you don't use vgo
     - go mod tidy
     # you may remove this if you don't need go generate
     - go generate ./...
+    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+    - /bin/bash -c './scripts/builld-sign-release-images.sh'
```

Comment:
I think we want to build and release the binaries first, then build the image. Is that something we should do in goreleaser.yaml, or in a step of the GH workflow?

Reply:
I agree with Jason, don't you think doing this in GH workflow would be more proper than having a bash script?

Reply:
this makes me remember one thing, let me test it 😄